### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Razzeee**
Fixes some typos in the PR template

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ### Description
 <!--- Provide a short summary of submitted add-on in case it's a new addition. -->
 <!--- If it's plugin update only highlight biggest changes if needed. -->
-<!--- Make sure you follow the checklist below before finalising your pull-request. -->
+<!--- Make sure you follow the checklist below before finalizing your pull-request. -->
```
Comment (MartijnKaijser) [inline PULL_REQUEST_TEMPLATE.md]: Why did you change this?  finalising  is British English

**Comment by hudokkow** [comment]
mention-bot works here but not on core repo?

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Razzeee) [inline PULL_REQUEST_TEMPLATE.md]: looked like an error to me, if you want me to change it back just say so.
